rename-icon.patch

```diff
@@ -0,0 +1,24 @@
+Author: Alberto Garcia <berto@igalia.com>
+Subject: Flatpak renames the Fuse icon so update the code accordingly
+diff --git a/ui/gtk/gtkui.c b/ui/gtk/gtkui.c
+index 835b74e5..c11f5083 100644
+--- a/ui/gtk/gtkui.c
++++ b/ui/gtk/gtkui.c
+@@ -169,7 +169,7 @@ ui_init( int *argc, char ***argv )
+   gtkui_window = gtk_window_new(GTK_WINDOW_TOPLEVEL);
+ 
+ #ifdef FUSE_ICON_AVAILABLE
+-  gtk_window_set_icon_name( GTK_WINDOW( gtkui_window ), "fuse" );
++  gtk_window_set_icon_name( GTK_WINDOW( gtkui_window ), "net.sf.fuse_emulator" );
```

Comment:
you could just have added `copy-icon: true` next to the `rename-icon`.

Reply:
I actually had a quick look at the documentation because I expected that there would be something like that and I didn't see it :) I saw it now that I checked again.

Anyway, I can do that in the next upload. Thanks!